backend/src/db/migrations/20230910205051_add_media_data_to_meditation_entries.ts

```diff
@@ -0,0 +1,25 @@
+import { type Knex } from 'knex';
+
+const TABLE_NAME = 'meditation_entries';
+
+const ColumnName = {
+  ID: 'id',
+  MEDIA_URL: 'media_url',
+  CONTENT_TYPE: 'content_type',
+} as const;
+
+function up(knex: Knex): Promise<void> {
+  return knex.schema.alterTable(TABLE_NAME, (table) => {
+    table.text(ColumnName.MEDIA_URL).notNullable().defaultTo('');
```

Comment:
let's drop all entities before instead of defaultTo, because if they don't have url, we don't need them
```suggestion
    table.text(ColumnName.MEDIA_URL).notNullable();
```

Reply:
Done